src/java/ognl/OgnlRuntime.java

```diff
@@ -899,12 +913,53 @@ public static Object invokeMethod(Object target, Method method, Object[] argsArr
                 }
             }
 
-            result = method.invoke(target, argsArray);
+            result = invokeMethodInJavaSandbox(target, method, argsArray);
         }
 
         return result;
     }
 
+    private static Object invokeMethodInJavaSandbox(Object target, Method method, Object[] argsArray) throws InvocationTargetException, IllegalAccessException {
```

Comment:
I would name this method as `invokeMethodInSandbox`

Reply:
As I use and honor [JDK's internal default sandbox model](https://docs.oracle.com/javase/7/docs/technotes/guides/.../security-spec.doc1.html) I named such that to refer reader to JDK sandbox (not our own or third-party sandbox impel).